### DISCUSSION TITLE
Lost a drive with recordings - how can I easily clean up library?

### DISCUSSION THREAD

**Discussion by bnflaherty** [OpenSageTV/Community#92] [Q&A]
Hey folks:

Seems I lost close to 1.5TB of recordings due to a drive failure.
Sage still thinks they exist but I can see which ones are "bad" as the comskip designation is missing.

Any easy way to re-scan to remove the invalid entries?

Thanks!

**Comment by Ugadata** [comment]
I believe if you replicate the Drive and Folder structure of the failed drive, Once Sage "sees" the old location (and that it is blank) Sage will remove the entries that are no longer available.  Although you could force a scan, Sage will eventually do a scan on its own, but it needs to "see" the old location in order for this to happen.
